mokapot/tabular_data/format_chooser.py

```diff
@@ -4,21 +4,46 @@
 import numpy as np
 from typeguard import typechecked
 
-from mokapot.tabular_data import (
+from mokapot.tabular_data.base import (
     BufferType,
-    BufferedWriter,
     ColumnMappedReader,
     ColumnSelectReader,
+    TabularDataReader,
+    TabularDataWriter,
+    DataFrameReader,
+)
+from mokapot.tabular_data.streaming import (
+    BufferedWriter,
+)
+from mokapot.tabular_data.csv import (
     CSVFileReader,
     CSVFileWriter,
+)
+
+from mokapot.tabular_data.parquet import (
     ParquetFileReader,
     ParquetFileWriter,
-    SqliteWriter,
-    TabularDataReader,
-    TabularDataWriter,
+)
+from mokapot.tabular_data.sqlite import SqliteWriter
+from mokapot.tabular_data.traditional_pin import (
+    is_traditional_pin,
+    read_traditional_pin,
 )
 
-CSV_SUFFIXES = [".csv", ".pin", ".tab", ".csv"]
+CSV_SUFFIXES = [
```

Comment:
I absolute agree. I just don't see a better way, as those other extensions are already out their in the wild.

Reply:
I don't recall any tool that generates a tab delimited .csv of the top of my head. Do you happen to have an example? (I wont deal with it in this PR but in the future we could split csv-tsv formats internally)

Reply:
Sorry, you're right. I somehow misread your initial comment. Yes, since we really never have "comma-separated" values anywhere, why not get completely rid of it and replace "comma separated/CSV" with "tab separated/TSV" everywhere. 

For the record: when I started on this code base, it was something with "comma separated" everywhere, but a separator variable `sep` was passed around, which was always set to "\t". I got rid of all the explicit file reading/writing stuff and moved that into the readers/writers, set the separator (I think) unconditionally to "\t", but did not rename the variables/classes. So: my bad ;)